src/Collection.php

```diff
@@ -1517,4 +1517,36 @@ public function stats()
             'collstats' => $this->getName(),
         ));
     }
+
+    /**
+     * Rename collection x to y, with drop previously created target or not option
+     * Note: if dropTarget is false and collection existent, then command will fail
+     *
+     * @param string $target Target name of collection
+     * @param bool $dropTarget Whether to drop previously created collection
+     * @return bool
+     * @throws \Sokil\Mongo\Exception
+     * @throws \MongoException
+     */
+    public function renameCollection($target, $dropTarget = false)
+    {
+        // create admin db instance
+        $adminDb = new Database($this->getDatabase()->getClient(), 'admin');
+
+        // getting current context db name
+        $sourceDbName = $this->getDatabase()->getName();
+
+        // rename current collection to target
+        $response = $adminDb->executeCommand(array(
+            'renameCollection' => $sourceDbName . '.' . $this->getName(),
+            'to'               => $sourceDbName . '.' . $target,
+            'dropTarget'       => $dropTarget,
+        ));
+
+        if ($response['ok'] === 1.0) { // renamed successfully
```

Comment:
if you throw exception on error, there are no reason to return true on success

Reply:
Fixed